src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/pay-pal/pay-pal.ts

```diff
@@ -0,0 +1,54 @@
+import PaymentForm from '../payment-form/payment-form';
+
+export default class PayPal extends PaymentForm {
+    form: HTMLFormElement;
+    braintreePayPalMethod: HTMLElement;
+
+    readonly braintreePayPal: string = 'braintreePayPal';
+    readonly PayPalAccount: string = 'PayPalAccount';
+
+    protected readyCallback(): void {
+        this.form = <HTMLFormElement>document.getElementById(`${this.formId}`);
+        this.braintreePayPalMethod = <HTMLElement>this.form.querySelector(`.${this.jsName}__method`);
+
+        super.readyCallback();
+    }
+
+    protected errorHandler(error: any) {
+        const paymentMethod = this.currentPaymentMethodValue;
+        const errorContainer = <HTMLElement>this.querySelector(`.${this.jsName}__error`);
+
+        this.emptyErrorContainer();
+
+        if (paymentMethod === this.braintreePayPal) {
+            return errorContainer.innerHTML = this.errorTemplate(error.message);
+        }
+
+        return this.submitForm();
+    }
+
+    protected paymentMethodHandler(response: any) {
```

Comment:
Please create this method implementation inside PaymentForm. The only difference - check of `paymentMethod` and `response.type`.